client/src/partials/templates/breadcrumb.tmpl.html

```diff
@@ -2,10 +2,10 @@
   <div class="bhima-title">
 
     <ol class="headercrumb">
-      <li ng-repeat="p in vm.bcPaths" id="{{ p.id }}" data-method="{{ p.dataMethod }}"  ng-class="{ 'static' : !$last, 'title' : $last }">
+      <li ng-repeat="p in vm.bcPaths" data-method="{{ p.dataMethod }}"  ng-class="{ 'static' : !$last, 'title' : $last }">
         <span ng-hide="p.link">{{ p.label | translate }}</span>
         <span ng-show="p.link">
-          <a ng-href="{{ p.link }}" href>{{ p.label | translate }}</a>
+          <a ng-href="{{ p.link }}" id="{{ p.id }}" href>{{ p.label | translate }}</a>
```

Comment:
I would be much more comfortable if this component prefixed the `id`s with some string, like `bc-` or `crumb-btn` or something.  That way, `id`s are guaranteed to be unique on a page.  I can easily imagine a programmer using numeric `id`s that would be used elsewhere in the module template.